### ISSUE TITLE
[BUG] osm_pass1 dying during parallel operation

### ISSUE THREAD

**Issue by hyperknot** [onthegomap/planetiler#1015]
**Describe the bug**
If I start a planet run, and while it's working I start a monaco run, the monaco run terminates.

**To Reproduce**
planetiler version: cf49b86

Start both processes with this logic:

```py
    java_memory_gb = 30 if area == 'planet' else 1

    command = [
        'java',
        f'-Xmx{java_memory_gb}g',
        '-jar',
        config.planetiler_path,
        f'--area={area}',
        '--download',
        '--download-threads=10',
        '--download-chunk-size-mb=1000',
        '--fetch-wikidata',
        '--output=tiles.mbtiles',
        '--nodemap-type=array',
        '--storage=mmap',
        '--force',
    ]

    if area == 'planet':
        command.append('--bounds=planet')
```

Logs:
[planetiler.out.txt](https://github.com/user-attachments/files/16995176/planetiler.out.txt)
[planetiler.err.txt](https://github.com/user-attachments/files/16995175/planetiler.err.txt)

**Expected behavior**
Parallel processes shouldn't effect each other.

**Environment (please complete the following information):**

- Hardware: Physical server with 128 GB memory
- OS: Ubuntu 22.04
- Java version and distribution: 
openjdk version "21.0.4" 2024-07-16
OpenJDK Runtime Environment (build 21.0.4+7-Ubuntu-1ubuntu222.04)
OpenJDK 64-Bit Server VM (build 21.0.4+7-Ubuntu-1ubuntu222.04, mixed mode, sharing)
- Maven version: ?

**Comment by msbarry** [comment]
How was the monaco run kicked off? Are they running in different subdirectories? The temp files will collide otherwise. Also for a small extract like monaco you should use `--nodemap-type=sortedtable` which will take much less time and storage.

**Comment by hyperknot** [comment]
No, monaco definitely works fine on it's on. My testing cron setup was to run monaco once per hour and planet once per day. It was very obvious that monaco only failed when planet was running on the same host.

**Comment by msbarry** [comment]
Just following up here, have you seen this at all since switching to sortedtable for monaco?

**Comment by hyperknot** [comment]
No, I made sure that they are not running at the same time. I might just move back Monaco to a conflicting time, but my worry is what if the planet process get terminated.

So basically I'd only do this for debugging, I'm actually happy that they are running in non-conflicting times now.
